tests/src/test/scala/ch/renku/acceptancetests/workflows/FlightsTutorial.scala

```diff
@@ -26,6 +26,9 @@ trait FlightsTutorial extends GivenWhenThen with Matchers with ScalatestMatchers
     When("the user clones the project locally")
     console %> c"git clone ${projectUrl.addGitCredentials} $projectFolder"
 
+    And("migrates the project")
```

Comment:
we should make sure that this is removed when the tests are fixed and everything is working consistently